.travis.linux.trusty.sh

```diff
@@ -0,0 +1,21 @@
+#! /bin/bash
+
+URL=ftp://ftp.simplesystems.org/pub/libpng/png/src/libpng16/libpng-1.6.23.tar.gz
+FILE=libpng-1.6.23.tar.gz
```

Comment:
Would it make more sense to include libpng as a submodule? Then it's easily just compiled into the build, rather than done separately. Or, even do it this way: in CMake, run `FindLibPNG`, and if it doesn't find it, then include the submodule version.